src/UI/components/about/aboutTeam.tsx

```diff
@@ -1,22 +1,137 @@
+                  padding: '10px',
+                  paddingLeft: '15px',
+                  paddingRight: '15px',
+                  cursor: 'pointer',
+                }}
+                onClick={() => setSelectedTeamMember(teamMember)}
+              >
+                <div style={{ display: 'flex', justifyContent: 'center' }}>
+                  <Avatar
+                    sx={{
+                      height: 100,
+                      width: 100,
+                      border:
+                        teamMember === selectedTeamMember
+                          ? '5px solid green'
+                          : '',
+                    }}
+                    alt={teamMember.name}
+                    src={teamMember.imageURL}
+                  />
+                </div>
+                <Box
+                  sx={{
+                    fontWeight: 'bold',
+                    color: 'primary.main',
+                    fontSize: 16,
+                    textAlign: 'center',
+                  }}
+                >
+                  {teamMember.name}
+                </Box>
+                <Box sx={{ fontWeight: 'Medium', textAlign: 'center' }}>
+                  {teamMember.location}
+                </Box>
+                <Box
+                  sx={{
+                    fontSize: isMatch ? '9px' : '12px',
+                    textAlign: 'center',
+                  }}
+                >
+                  {teamMember.position}
+                </Box>
+              </div>
+            ))}
+          </div>
+        </Grid>
+        {selectedTeamMember ? (
+          <Grid item xs={12} md={3}>
+            <Box
+              sx={{
+                border: 3,
+                color: 'primary.main',
+                padding: 2,
+                borderRadius: 5,
+              }}
+            >
+              <Box
+                sx={{
+                  fontWeight: 'bold',
+                  color: 'primary.main',
+                  fontSize: 20,
+                  justifyContent: 'space-between',
+                }}
+                data-testid="teamListbox"
+              >
+                {selectedTeamMember.name}
+                <Button
+                  onClick={() => setSelectedTeamMember(!selectedTeamMember)}
+                  size="small"
+                  sx={{
+                    float: 'right',
+                    margin: 0,
+                    padding: 0,
+                    paddingLeft: 0,
+                    minHeight: 0,
+                    minWidth: 0,
+                  }}
+                >
+                  {<HighlightOffOutlinedIcon sx={{ color: 'gray' }} />}
+                </Button>
+              </Box>
+              <Box sx={{ paddingTop: 3, minHeight: 130 }}>
+                <Typography variant="body1">
+                  {selectedTeamMember['description']}
+                </Typography>
+              </Box>
+            </Box>
+          </Grid>
+        ) : null}
```

Comment:
Instead of this null option, you can change line 91 to be `{selectedTeamMember && `